src/Covid19Dashboard/index.jsx

```diff
@@ -110,23 +112,52 @@ class Covid19Dashboard extends React.Component {
   render() {
```

Comment:
since we do not want the charts in prod right away, we need a flag to enable/disable them. for now it can just be a `const enableCharts=false` line in the `render` function. you don't have to do it but this comment is a reminder for me to do it later if you don't :)

Reply:
I'm not sure how to implement this. To make it nice, the CSS should be changed also. I can take a look.

Reply:
yeah, did you have time to play with this?